src/libraries/Microsoft.PowerFx.Core/Public/ErrorKind.cs

```diff
@@ -39,6 +39,7 @@ public enum ErrorKind
         NotApplicable = 27,
         Timeout = 28,
         ServiceUnavailable = 29,
+        InvalidJson = 30,
```

Comment:
>InvalidJson = 30, [](http://example.com/codeflow?start=8&length=17)

Should ParseJSON() use this?

Reply:
The objectives are too different, I wouldn't try merging them

Reply:
I don't think so, this is really for connectors, when the result coming back from the connector (directly or via APIM) is invalid in a way or another (invalid Json like missing '}', or logically invalid like receiving a string instead of a double)